scripts/log-session.py

```
@@ -0,0 +1,334 @@
+    if files:
+        print(f'\nDetected {len(files)} modified files:')
+        for f in files[:5]:
+            print(f'  - {f}')
+        if len(files) > 5:
+            print(f'  ... and {len(files) - 5} more')
+        if input('Use these files? [Y/n]: ').strip().lower() == 'n':
+            files = []
+
+    detected_task_id = detect_task_id()
+    if detected_task_id:
+        task_id = prompt_input(f'\nTask ID', detected_task_id)
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)